internal/testplanet/planet.go

```diff
@@ -76,9 +76,10 @@ type Config struct {
 	StorageNodeCount int
 	UplinkCount      int
 
-	Identities      *testidentity.Identities
-	IdentityVersion *storj.IDVersion
-	Reconfigure     Reconfigure
+	Identities         *testidentity.Identities
+	IdentityVersion    *storj.IDVersion
+	Reconfigure        Reconfigure
+	UsePeerCAWhitelist bool
```

Comment:
Why this needs to be added here? For testing it should be possible to manipulate this param with `Reconfigure`

Reply:
Nice, I forgot about that